iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -301,7 +301,6 @@ class ConversationListViewController: InputHandlerViewContoller, ConversationLis
                 guard !Task.isCancelled else { return }
 
                 self?.messageInputController.resetExpression()
-                self?.messageInputController.resetDeliveryType()
```

Comment:
Delivery type is no longer selected before hand, so it doesn't need to be reset.